src/raytracer/Raytracer.m

```diff
@@ -181,24 +207,24 @@
                             switchMaterial, vtx, vrx, ...
                             paraCfgInput.switchDiffuseComponent,...
                             paraCfgInput.switchQDModel,...
-                            paraCfgInput.inputScenarioName(10:end),...
+                            scenarioName,...
                             paraCfgInput.carrierFrequency,...
                             paraCfgInput.diffusePathGainThreshold,...
-                            paraCfgInput.reflectionLoss,...
-                            'qTx', QTx, 'qRx', QRx);
+							paraCfgInput.reflectionLoss, ...
+                            'rotTx', QTx.angle, 'rotRx', QRx.angle);
                         
                         nMpc = size(multipathTemporary,1);
                         %Store MPC
                         if paraCfgInput.switchSaveVisualizerFiles &&...
-                                nMpc > 0                            
-                                multipath1 = multipathTemporary(:,...
-                                    2:end); %Discard reflection order column
-                                Mpc{iterateTx,iteratePaaTx,...
-                                    iterateRx,iteratePaaRx, ...
-                                    iterateOrderOfReflection+1, iterateTimeDivision+1} =multipath1;                            
+                                nMpc > 0
```

Comment:
is would but the two on the same line since it's quite short, or maybe add some brackets to make it clearer